xapian-applications/omega/configure.ac

```diff
@@ -246,6 +246,16 @@ AC_SEARCH_LIBS([socketpair], [xnet],
 dnl fnmatch() is used to match ignore pattern.
 AC_CHECK_FUNCS([fnmatch])
 
+dnl check if poppler is available.
+am_save_LIBS="$LIBS"
+AC_LIB_APPENDTOVAR([LIBS], [-lpoppler-cpp])
+AC_TRY_LINK([#include <poppler/cpp/poppler-version.h>],
+            [poppler::version_string()],
+            [AC_DEFINE(HAVE_POPPLER, 1, [Define HAVE_POPPLER if the labrary is available]), lib_poppler=true],
```

Comment:
"labrary" should be "library"

Reply:
And that `,` before `lib_poppler=true` is wrong - that should be a newline or a `;` - this expands to shell code.

Reply:
Okay, I do not have much experience with this type of tool. I'm afraid I might make these kinds of mistakes.